test/test_asn1.rb

```diff
@@ -566,6 +566,19 @@ def test_decode_constructed_overread
     assert_equal 17, ret[0][6]
   end
 
+  def test_constructive_each
+    data = ['test','data']
+    seq = OpenSSL::ASN1::Sequence.new data
```

Comment:
'data' should be an array of OpenSSL::ASN1::ASN1Data instances rather than of String.